ansible/20_tempest_ocp.yaml

```diff
@@ -55,7 +55,7 @@
         oc create configmap tempest-config --from-file {{ base_path }}/container_tempest_cfg
       environment:
         PATH: "/usr/local/bin:{{ ansible_env.PATH }}"
-        KUBECONFIG: "{{base_path}}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"
+        KUBECONFIG: "{{ kubeconfig_path }}"
```

Comment:
we have already {{ kubeconfig }} , like e.g. https://github.com/openstack-k8s-operators/dev-tools/blob/master/ansible/install_keystone.yaml#L14 . Isn't that what you want?

Reply:
It's definitely possible.  I may have misunderstood what was already there.  Let me take a look.

Reply:
So `"{{base_path}}/dev-scripts/ocp/{{ ocp_cluster_name }}/auth/kubeconfig"` seems to be a different path than the `kubeconfig` variable created by https://github.com/openstack-k8s-operators/dev-tools/blob/9ce00a6c56f899f82599ceb7c08f6ad443e7f129/ansible/roles/oc_local/tasks/main.yaml#L15.  That being said, they're both ultimately the same kubeconfig (I believe, just existing in two different places), so I think you're right and my additional variable is superfluous.

Reply:
I haven't ported this playbook to use oc_local, so it presumably only works if you execute it on the ocp host. We should probably do that rather than this fixup, tbh.